src/main/java/ua/softserveinc/tc/service/impl/RoomServiceImpl.java

```diff
@@ -169,14 +169,47 @@ public Boolean isPossibleUpdate(BookingDto bookingDto) {
     }
 
     /**
+     * The method finds the maximum people in the room for period of time
+     * from dateLo to dateHi.
+     * @param dateLo start of period
+     * @param dateHi end of period
+     * @param bookings all reserved bookings in the time period
+     * @return The maximum number of people that are simultaneously in the room
+     */
+
+    private int maxRangeReservedBookings (Date dateLo, Date dateHi, List<Booking> bookings) {
+        if(dateLo == null)
+            throw new NullPointerException("dateLo is null");
+        else if(dateHi == null)
+            throw new NullPointerException("dateHi is null");
+        else if(bookings == null)
```

Comment:
this is wrong
at least you can do all these checks in one if statement
and do not throw Exceptions - if you have some nulls - just handle them.